api-tests/apply_route_test.go

```diff
@@ -403,10 +408,12 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"recommended", "1.7.0", nil, "4.4.3-5"},
 		{"recommended", "1.6.0", nil, "4.4.2-4"},
 		{"recommended", "1.5.0", nil, "4.2.8-8"},
+		{"recommended", "1.16.0", &v50, "6.0.15-12"},
```

Comment:
it cannot be 6.0.15, it needs to be some 5.0 version since the assertion is for 5.0